app/views/contacts/new.html.haml

```diff
@@ -0,0 +1,31 @@
+.panel.panel-default
+  .panel-heading.text-center
+    %h3 Feedback
+  - if user_signed_in?
+    .panel.panel-body
+      %p Scoreboard welcomes your feedback as a BETA tester.
+      = simple_form_for @contact do |f|
+        .form-group
+          = f.label "Name"
+          = f.text_field :name, required: true, class: "form-control"
+        .form-group
+          = f.label "Email"
+          = f.email_field :email, required: true, class: "form-control"
+        .form-group
+          = f.label "What's your feedback?"
+          = f.text_area :message, required: true, class: "form-control", size: "1x3", placeholder: "Leave a comment..."
+        .hidden
+          = f.input :nickname, hint: 'Leave this field blank!'
+        %div
+          = f.submit 'Send message', class: "btn btn-primary"
+  - else
+    .panel.panel-body.text-center
+      %br
+      %p To see this page, you must be signed in!
+      %p
```

Comment:
P doesn't need to be here or are you trying to group stuff into a paragraph block